src/mlpack/bindings/go/print_go.cpp

```diff
@@ -81,16 +81,15 @@ void PrintGo(const util::ProgramDoc& programInfo,
   cout << endl;
 
   // Then we must print the import of the gonum package.
-  cout << "import (" << endl;
-  cout << "  " << "\"gonum.org/v1/gonum/mat\" " << endl;
-  for (size_t i = 0; i < inputOptions.size(); ++i)
+  for (ParamIter it = parameters.begin(); it != parameters.end(); ++it)
   {
-    const util::ParamData& d = parameters.at(inputOptions[i]);
-    size_t indent = 2;
-    CLI::GetSingleton().functionMap[d.tname]["ImportDecl"](d,
-        (void*) &indent, NULL);
+    const util::ParamData& d = it->second;
+    if ((d.cppType).compare(0, 6, "arma::") == 0)
+    {
+      std::cout << "import \"gonum.org/v1/gonum/mat\" " << std::endl;
+      break;
+    }
```

Comment:
Hmm, I think that there are no cases where we have bindings that don't have Armadillo parameters, but it's nice to be specific like this.  Are you sure that it's okay to avoid importing Gonum even if some of the mlpack support functions take Gonum objects as parameters?

Reply:
I think so hmm_train dosen't use armadillo parameter.

Reply:
You're right, nice call; that one takes in a string.  However I can see we're not currently building the HMM bindings for Go.  Maybe we should enable that and by building those we'll test to make sure avoiding importing Gonum is okay?  Or maybe there was some other issue for why we don't build the HMM bindings; I'm not sure.

Reply:
Sorry I didn't informed but previously with `GMM`, I had also added Go-bindings for `HMM` too.

Reply:
Awesome, great to add those too!